watertap/costing/watertap_costing_package.py

```diff
@@ -29,6 +29,7 @@
 from idaes.models.unit_models import Mixer
 
 from watertap.unit_models import (
+    Ultraviolet0D,

```

Comment:
Typically, we arrange the sequence alphabetically, but since there are not so many detailed models, so people just add them following the last one. I add it at the first just for avoiding potential merging conflicts because I believe there are other detailed model PRs (e.g. yours and Kurby's). If we all add it at the last, then we will have merging conflicts. But I agree with you that we can follow the sequence in the **init** file at present.

Reply:
@TimBartholomew @adam-a-a Is it worth creating an issue for standardizing the order of the unit models when presented as a list in various locations? See `watertap/docs/technical_reference/unit_models/index.rst`, `watertap/watertap/unit_models/__init__.py`, and `watertap/watertap/costing/watertap_costing_package.py -> imports` as examples that all appear differently. This is extremely minor and more ease of use I would think, but may be worth converting to alphabetical as the list gets longer and as @luohezhiming said, to more consistently avoid merge conflicts with new models' PRs. Currently it seemed as though the list had the newest models at the end of the list but that has gotten twisted as the aforementioned issues occur.

Reply:
We should generally do things alphabetically. This, along with the way `black` formats the code, will very often avoid merge conflicts.